website_sale_brand_catalogs/models/website.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class Website(models.Model):
+
+    _inherit = "website"
+    brand_bpp = fields.Integer(default=20, string="Number of brands")
```

Comment:
You removed the wrong blank line. The one between private attributes and fields should actually be present, the one to remove is between class declaration and private attributes.

```
class ResPartner(models.Model):
    _inherit = "res.partner"  # no blank lines above

    active = fields.Boolean()  # One blank line above
```

Same in product brand